### ISSUE TITLE
Multiple Entries of same Host Overrides not possible

### ISSUE THREAD

**Issue by uffsalot** [jaredhendrickson13/pfsense-api#73]
Currently it is not possible to create multiple host overides for a specific host/domain combination. However, this would be very useful for a dual-stack configuration or simple dns loadbalancing. The pfsense UI allow such a configuration.

```
{
    "status": "bad request",
    "code": 400,
    "return": 2010,
    "message": "Unbound host override already exists",
    "data": []
}
```

Maybe it makes sense to rebuild the check to check the combination of host override and IP address. This would prevent exact duplicates, but also allow different IP addresses on a single host override.

**Comment by jaredhendrickson13** [comment]
Hey!

Thanks for bringing this up. This feature is not currently available in pfSense 2.4.x which is why it's not currently implemented in the API. I am planning on implementing pfSense 2.5+ exclusive features like these closer towards the EOL date for pfSense 2.4. I will keep this item backlogged for now.

Thanks!

**Comment by uffsalot** [comment]
Hi,
thanks for the head up. But I don't understand what you mean with "This feature is not currently available in pfSense 2.4.x"
I am using 2.4.5-p1 and it's working fine (Actually since I started using pfsense which was a few years ago)

![2021-02-18 00_10_55-FWPFS01 biocrafting net - Services_ DNS Resolver_ General Settings](https://user-images.githubusercontent.com/1371511/108280176-178f0900-717e-11eb-85c1-5e59f303babc.png)

```
root@JBRDEB01:~# host aptdeb01.int.biocrafting.net
aptdeb01.int.biocrafting.net has address 10.10.10.101
aptdeb01.int.biocrafting.net has IPv6 address 2a0a:51c0:7:1010::101
root@JBRDEB01:~# host apt.int.biocrafting.net
apt.int.biocrafting.net has address 10.10.10.101
apt.int.biocrafting.net has IPv6 address 2a0a:51c0:7:1010::101
```

Or did I misunderstood something?

**Comment by jaredhendrickson13** [comment]
My apologies I misunderstood the issue. I was thinking you were referring to multiple IPv4 or IPv6 addresses for a single host override. In this case you are correct, you are allowed to configure one IPv4 and one IPv6 host override per host/domain combination which is not being allowed by the API currently. This should be an easy fix. I can have a fix for this in the next release. Sorry for the confusion.

Thanks!
